main/modes/games/ultimateTTT/ultimateTTTcpuPlayer.c

```diff
@@ -1102,6 +1182,77 @@ static uint16_t analyzeSubgame(const tttPlayer_t subgame[3][3], tttPlayer_t play
     return 0;
 }
 
+static bool analyzeMove(const tttSubgame_t subgames[3][3], const move_t* move, tttPlayer_t player,
+                        moveAnalysis_t* result)
+{
+    tttSubgame_t board[3][3];
+    if (subgames[move->subX][move->subY].winner != TTT_NONE
+        || subgames[move->subX][move->subY].game[move->cellX][move->cellY] != TTT_NONE)
+    {
+        // Impossible move!
+        return false;
+    }
+
+    memcpy(board, subgames, sizeof(board));
+
+    const tttPlayer_t opponent = (player == TTT_P1) ? TTT_P2 : TTT_P1;
+    tttPlayer_t turn           = player;
+    move_t lastMove            = {0};
+    memcpy(&lastMove, move, sizeof(move_t));
+    int moveNum = 0;
+
+    {
```

Comment:
Scoped for fun? Did this have logic at some point?

Reply:
Oooh yeah it did, originally there was a loop here for some reason

Reply:
Stray braces don't bother me much if they don't bother you

Reply:
Eh I'll fix this and the crazy comment, together they bother me just enough

Reply:
I saw that comment and opted not to call you out on it. I don't like getting too nitpicky on CRs.